Page/Wishlist/wishlist.html

```diff
@@ -18,7 +18,7 @@
     <header>
         <nav class="nav-container-box">
             <div class="nav-logo"><a href="/../index.html">Scentcy <i class="fa-solid fa-spray-can-sparkles"></i></a></div>
-            <input type="search" placeholder="   Search for items here" class="nav-input">
+            <input type="search" placeholder=" Search for items here" class="nav-input">
```

Comment:
There's a leading space left here in placeholder. You can remove it.

Reply:
okay